package.json

```diff
@@ -10,14 +10,23 @@
     "WordPress",
     "editor"
   ],
+  "files": [
+    "index.php",
+    "post-content.js",
+    "editor/build",
+    "i18n/build",
+    "element/build",
+    "blocks/build"
+  ],
   "scripts": {
     "test-unit": "cross-env NODE_ENV=test webpack && mocha build --require bootstrap-test.js",
     "build": "cross-env BABEL_ENV=default NODE_ENV=production webpack",
     "gettext-strings": "cross-env BABEL_ENV=gettext webpack",
     "lint": "eslint .",
     "dev": "cross-env BABEL_ENV=default webpack --watch",
     "test": "npm run lint && npm run test-unit",
-    "ci": "concurrently \"npm run build\" \"npm test\""
+    "ci": "concurrently \"npm run build\" \"npm test\"",
+    "build-plugin": "rm -f gutenberg.zip && npm install && npm run build && cat \"$(npm pack)\" | tar xzf - && cd package && zip -r ../gutenberg.zip * && cd - && rm -rf package gutenberg*.tgz"
```

Comment:
Do we need to clean up the zip before starting the build? I didn't realize `zip` _adds to_ an archive by default, but does it do so as well with the `-r` flag? Thinking it could shave a few characters to include this in the final `rm` of the command.

Also, "pre" scripts are a nice way to separate preparation tasks from the script itself:

https://docs.npmjs.com/misc/scripts

`install` seems to fall under this; `build` not so much. But if we renamed, maybe this could be written as:

```
"prepackage-plugin": "npm install && npm run build",
"package-plugin": "cat ..."
```

This does add another line/script, but do you think it helps clarify the packaging behavior?